Directory.Build.props

```diff
@@ -77,6 +77,11 @@
     <LicenseUrl>https://github.com/dotnet/runtime/blob/master/LICENSE.TXT</LicenseUrl>
     <RuntimeIdGraphDefinitionFile>$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'pkg', 'Microsoft.NETCore.Platforms', 'runtime.json'))</RuntimeIdGraphDefinitionFile>
     <LicenseFile>$(RepoRoot)LICENSE.TXT</LicenseFile>
+    <PackageLicenseFile>$(RepoRoot)LICENSE.TXT</PackageLicenseFile>
```

Comment:
We have 2 properties LicenseFile on line 79 and packageLicenseFile. Can we just keep the LicenseFile ?
PackageLicenseFile is a deprecated property in nuget pack and cannot be set